analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/analog-minimalist-hour-ambient.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="800"
+   height="800"
+   viewBox="0 0 211.667 211.667"
+   version="1.1"
+   id="svg756"
+   sodipodi:docname="analog-minimalist-hour-ambient.svg"
```

Comment:
None of this inkscape stuff is relevant to the watch.  You should run these SVG files through `svgo` or export as "Optimized SVG" from inkscape to strip out irrelevant metadata.